app/sections/view-native-mobile/macro.njk

```diff
@@ -16,41 +16,41 @@
       questionId: 'operating-systems-description',
       questionTitle: 'Additional operating system information',
       layout: 'vertical',
-      innerComponent: viewQuestionDataText({
-        questionId: 'operating-systems-description',
-        questionData: params.section.sections['native-mobile-operating-systems'].answers['operating-systems-description']
+      innerComponent: viewDataText({
+        dataTestIdIdentifier: 'view-question-data-text-operating-systems-description',
```

Comment:
can this be reverted back to the original? I don't think the whole thing is needed unless the shorter one is being ued elsewhere

Reply:
also remember when we change data-test-id tags then it has a knock on effect with the testers as their tests will break so I usually try to avoid changing them where I can

Reply:
We wanted to get rid of the concept of question from the viewDataText component and make them very generic going forward. And also any magic stuff we do with data-test-ids inside them so it's bit more obvious what the data-test-id is.

We need to revisit this later and see if we actually need to pass in the question-id at all so just `view-data-text` perhaps. But for now we are going for a like for like. So we don't break marketing pages or public browse

Reply:
the `data-text` part at least could be moved inside then

Reply:
why not give the client of the `viewDataText` component complete control as to what they want the data-test-identifier to be?

Reply:
if you want to do that then why bother having view-data as part of _every_ data test id you have given 😄